src/main/webapp/app/app.tsx

```diff
@@ -42,9 +42,8 @@ const App: React.FunctionComponent<IAppProps> = (props: IAppProps) => {
             pauseOnFocusLoss
           />
           <BetaSiteMessage />
-          {props.loadingAuth ? (
-            <LoadingIndicator isLoading size={LoaderSize.LARGE} center={true} />
-          ) : (
+          {props.loadingAuth && <LoadingIndicator isLoading size={LoaderSize.LARGE} center={true} />}
```

Comment:
@calvinlu3 I think this solution is a bit hacky, let me know how you think.

Reply:
This looks fine to me. What issue did rendering the AppRoutes cause?